app/views/layouts/default.php

```diff
@@ -6,7 +6,7 @@
     
     
     <link href="/bootstrap/css/bootstrap.min.css" rel="stylesheet">
-    
+    <link href="/bootstrap/css/bootstrap.css" rel="stylesheet">
```

Comment:
I have edited some designs in bootstrap. but it doesn't apply unless i change bootstrap.css itself. that's why I included that po.